internal/cli/kraft/cloud/scale/initialize/initialize.go

```diff
@@ -28,9 +28,9 @@ type InitOptions struct {
 	Client       kraftcloud.KraftCloud `noattribute:"true"`
 	CooldownTime time.Duration         `long:"cooldown-time" short:"c" usage:"The cooldown time of the config (ms/s/m/h)" default:"1000000000"`
 	Master       string                `long:"master" short:"i" usage:"The UUID or Name of the master instance"`
-	MaxSize      uint                  `long:"max-size" short:"M" usage:"The maximum size of the configuration" default:"10"`
+	MaxSize      int                   `long:"max-size" short:"M" usage:"The maximum size of the configuration" default:"10"`
 	Metro        string                `noattribute:"true"`
-	MinSize      uint                  `long:"min-size" short:"m" usage:"The minimum size of the configuration"`
+	MinSize      int                   `long:"min-size" short:"m" usage:"The minimum size of the configuration"`
```

Comment:
Not sure about the platform implementation, but wouldn't it be better to have this as `uint` to catch user errors?

Min/Max are both >0 and if we use `uint` cobra can catch (I think) if someone used a negative value.

What do you think? I'm fine either way

Reply:
We can yes, I don't have strong opinions for/against it.

The reason why I changed `uint` to `int` in the SDK is that the API accepts `int` (to be clarified), and I would like the validation to remain as much as possible **on the API side** to avoid drifts in client code.